pydis_site/apps/api/viewsets/bot/bumped_thread.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework.mixins import (
+    CreateModelMixin, DestroyModelMixin, ListModelMixin, RetrieveModelMixin
+)
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot import BumpedThread
+from pydis_site.apps.api.serializers import BumpedThreadSerializer
+
+
+class BumpedThreadViewSet(
+    GenericViewSet, CreateModelMixin, DestroyModelMixin, RetrieveModelMixin, ListModelMixin
+):
+    """
+    View providing CRUD (Minus the U) operations on threads to be bumped.
+
+    ## Routes
+    ### GET /bot/bumped-threads
+    Returns all BumpedThread items in the database.
+
+    #### Response format
+    >>> [
+    ...     {
+    ...         'thread_id': "941705627405811793",
+    ...     },
+    ...     ...
+    ... ]
+
+    #### Status codes
+    - 200: returned on success
+    - 401: returned if unauthenticated
+
+    ### GET /bot/bumped-threads/<thread_id:int>
+    Returns a specific BumpedThread item from the database.
+
+    #### Response format
+    >>> {
+    ...     'thread_id': "941705627405811793",
+    ... }
```

Comment:
Again, it's clumsy. Just return the value directly (that's still valid JSON by the way). There is only ever going to be one field, so there's no point in having an object with a field name.

Reply:
https://github.com/python-discord/site/pull/663/commits/3e9557056c06a39c077b76d718eb35b99a365711

Reply:
Ah, this commit will require tests since I added a custom retrieve, will fix